lib/explorer/query/sql.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Explorer.Query.Sql do
+  @moduledoc false
+  defstruct [:string]
+
+  @doc false
+  def new(sql_string) when is_binary(sql_string) do
+    %__MODULE__{string: sql_string}
+  end
+
+  @doc false
+  def to_lazy_series(%__MODULE__{string: sql_string}) do
+    aggregation? = true
+    dtype_out = :unknown
+
+    Explorer.Backend.LazySeries.new(
+      :sql,
+      [sql_string],
+      dtype_out,
+      aggregation?,
+      Explorer.PolarsBackend
```

Comment:
Maybe we want to fetch this from `Explorer.Backend.get()`. It will return exactly the same, but let's people use their backend. WDYT?

```suggestion
      Explorer.Backend.get()
```